packages/demos/stories/FormPage.tsx

```diff
@@ -18,7 +19,7 @@ export default () => {
 	const [selectedItems, setSelectedItems] = React.useState<any>([]);
 
 	return (
-		<div className="container-fluid">
+		<ClayLayout.ContainerFluid>
```

Comment:
Interestingly, this is would actually be `xl` because of the [defaults](https://github.com/liferay/clay/blob/master/packages/clay-layout/src/ContainerFluid.tsx#L20)